src/commands/Configuration/configure/automod/limit.ts

```diff
@@ -13,16 +13,17 @@ export default class extends SubCommand {
 		super(client, parent, {
 			name: "limit",
 			clearance: 75,
-			minArgs: 2,
-			maxArgs: 2,
-			argList: ["part:string", "limit:number"],
+			minArgs: 3,
+			maxArgs: 3,
+			argList: ["part:string", "limit:number", "timeout:number"],
 			usage: "<part> <limit>",
 		});
 	}
 
 	public async run(context: CommandContext, args: string[]): Promise<void | Message> {
 		const partArg = args[0];
 		const limit = Number(args[1]);
```

Comment:
maybe something should be added when you try to add a negative amount to limit, just a thought tho